chart/templates/secrets/fernetkey-secret.yaml

```diff
@@ -35,7 +35,7 @@ metadata:
       {{- toYaml . | nindent 4 }}
     {{- end }}
   annotations:
-    "helm.sh/hook": "pre-install"
+    "helm.sh/hook": "pre-install,pre-upgrade"
```

Comment:
if the name changes it will never be installed anymore :( - do you have any suggestion? maybe an option in values.yaml like reinstallFernetKey? which adds the pre-upgrade hook optional? otherwise we'll never be able to update the Secret and all the references of volume mounts will be broken. that was the reason why I added it here but I'm open for suggestions :) Or we can keep these secrets as they were with the old names?

Reply:
I don't have a good suggestion unfortunately. We've considered [lookup](https://helm.sh/docs/chart_template_guide/function_list/#lookup) but never used it because of the it's behavior with template (which things like helm-diff and argocd use). Welcome to the backcompat nightmare :(

It might be worth leaving comments so when we do get around to accepting breaking changes in preparation for major version 2 of the chart, we can address it then.

Reply:
If you're good with it.. I'll keep the secrets that use the pre-install hook for now "untouched" with the original names to avoid making this a real breaking change and will add a comment in those resources for the future to think how to address the renaming of those with the standard naming

Reply:
Sounds good, thanks.

Reply:
kept original names for fernet-key, redis-password and broker-url. Added a note in values.yaml just as a reference